packages/flutter/lib/src/gestures/resampler.dart

```diff
@@ -245,16 +253,21 @@ class PointerEventResampler {
         // therefore never produce `hover` events.
         if (position != _position) {
           final Offset delta = position - _position;
-          callback(_toMoveOrHoverEvent(event, position, delta,
-              _pointerIdentifier, sampleTime, wasDown, hadButtons));
+          eventBuilders.add((bool endOfBatch) =>
```

Comment:
nit (here and elsewhere): https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#consider-using--for-short-functions-and-methods:

> [...] only use ⇒ when everything, including the function declaration, fits on a single line.